feature/account/setup/src/test/kotlin/app/k9mail/feature/account/setup/domain/entity/ConnectionSecurityTest.kt

```diff
@@ -0,0 +1,36 @@
+package app.k9mail.feature.account.setup.domain.entity
+
+import org.junit.Test
+
+class ConnectionSecurityTest {
+
+    @Test
+    fun `should provide right default smtp port`() {
+        val connectionSecurity = ConnectionSecurity.all()
+
+        for (security in connectionSecurity) {
+            val port = security.toSmtpDefaultPort()
+
+            when (security) {
+                ConnectionSecurity.None -> assert(port == 587L)
+                ConnectionSecurity.StartTLS -> assert(port == 587L)
+                ConnectionSecurity.TLS -> assert(port == 465L)
+            }
+        }
```

Comment:
It feels strange to use runtime assertions in a test. This could be rewritten to:

```kotlin
@Test
fun `should provide right default smtp port`() {
    val mapping = ConnectionSecurity.all().associateWith { it.toSmtpDefaultPort() }

    assertThat(mapping).containsOnly(
        ConnectionSecurity.None to 587L,
        ConnectionSecurity.StartTLS to 587L,
        ConnectionSecurity.TLS to 465L,
    )
}
```

That would also catch changes to `ConnectionSecurity.all()`.